src/List.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import * as R from 'ramda';
+import PropTypes from 'prop-types';
+
+import Item from './Item';
+
+export default function List({ items, onClick }) {
+  if (R.isEmpty(items)) {
+    return <p>할 일이 없어요!</p>;
+  }
+
+  return (
+    <ol>
+      {items.map((item) => (
+        <Item
+          content={item.content}
+          onClick={() => onClick(item.id)}
+        />
+      ))}
```

Comment:
```suggestion
      {items.map(({ content, id }) => (
        <Item
          content={content}
          onClick={() => onClick(id)}
        />
      ))}
```
구조분해할당을 적용해도 좋을 것 같네요 :)

추가로 key 값 추가가 필요해보입니다!

참고: [https://reactjs.org/docs/lists-and-keys.html](https://reactjs.org/docs/lists-and-keys.html)

Reply:
`b38d1df`, `bbe7315`에서 수정했습니다.